src/tracing/errors.ts

```diff
@@ -21,6 +21,9 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
 
     // If the error is a network change, it should not be considered an exception.
     if (error.message.match(/underlying network changed/)) return null
+
+    // If the error is based on a user rejecting, it should not be considered an exception.
+    if (error.message.match(/user rejected transaction/)) return null
```

Comment:
Instead of re-implementing matching logic, can you replace this with `didUserReject` for x-wallet compatibility ([src](https://github.com/Uniswap/interface/blob/2ffc8a0bdfc66fc1372aa2f15cf6c64f4afdbf02/src/utils/swapErrorToUserReadableMessage.tsx#L12))?